src/parser/jobs/rdm/index.js

```diff
@@ -32,5 +32,15 @@ export default new Meta({
 	],
 
 	changelog: [
+		{
+			date: new Date('2019-07-14'),
+			Changes: () => <>Initial update of existing logic for 5.0 (no new stuff)</>,
+			contributors: [CONTRIBUTORS.LEYLIA],
+		},
+		{
```

Comment:
Can you flip these for me next time you add to the changelog? Once they get a bit long, it'll be easier to add to the top than scroll to the bottom to add there (plus it's in-line with the UI, which shows recent first).